Fieldset/styles.scss

```diff
@@ -2,7 +2,9 @@
 @import '../css/mixins/index';
 
 .fieldset {
+  background-color: white;
```

Comment:
Yea, we should avoid adding fixed background colors for things like this. If the border radius is changed through customization, this would not work anyway.